docs/versions.rst

```diff
@@ -16,17 +16,37 @@ AVM Version  PyTeal Version
 4            >= 0.8.0
```

Comment:
The AVM 7 PR renames this page from "TEAL Versions" to "AVM Versions". As part of the ABI user guide, I renamed it from "TEAL Versions" to "Versions".

I'd prefer to keep it as "Versions" since it talks about both PyTeal package versions and AVM versions.

Reply:
@jasonpaulos I changed the title via https://github.com/algorand/pyteal/pull/480/commits/1a9a0ee84be506bca6c16f64c50bf1128570d389 - Can you resolve if it looks good?

Reply:
Verbally agreed go to go - resolving.